### ISSUE TITLE
CoordConv layers

### ISSUE THREAD

**Issue by Higgs32584** [mit-ll/tornet#9]
Hi, can someone please provide an example on how to add CoordConv Layers to your Neural network?

Chat GPT seems to be struggling with all of the custom datasets, it would be very helpful If someone explained it, as it seems to be virtually impossible to get a high AUC without a CoordConv. Thank you

**Comment by tonyreisll** [comment]
The coordconv layer just concatenates coordinate information along the channel dimension before doing a convolution. This is done at multiple layers of the network so the coordinate channel's shape is also adjusted to match the shape of the other channels as their shape changes based on kernel size, stride and padding of previous convolution layers. Let me know if this is still a problem for you.

**Comment by Higgs32584** [comment]
Ok thank you, I see now. Thank you

is there anyway to unpack the dataset into .tfDataset that is faster than the currently available method? it took me nearly 2 days to create. thank you

**Comment by markveilletteLL** [comment]
There may be ways to parallelize the process, but yes, it does take a bit of time.   However the speed of the dataloader is well worth it IMO.
